apps/website/versioned_docs/version-v3.x/supported-networks/costs.md

```diff
@@ -0,0 +1,419 @@
+
+</details>
+
+<details>
+<summary>Deployment cost</summary>
+
+| Deployments                              | Min Gas | Max Gas | Avg Gas | % of Block Limit | USD (avg) |
+| ---------------------------------------- | ------- | ------- | ------- | ---------------- | --------- |
+| AnonAadhaarCheckerFactory                | -       | -       | 567621  | 1.9 %            | 0.057     |
+| AnonAadhaarPolicyFactory                 | -       | -       | 759390  | 2.5 %            | 0.0762    |
+| ConstantInitialVoiceCreditProxyFactory   | -       | -       | 379812  | 1.3 %            | 0.0381    |
+| EASCheckerFactory                        | -       | -       | 623304  | 2.1 %            | 0.0625    |
+| EASPolicyFactory                         | -       | -       | 690560  | 2.3 %            | 0.0693    |
+| ERC20CheckerFactory                      | -       | -       | 446429  | 1.5 %            | 0.0448    |
+| ERC20PolicyFactory                       | -       | -       | 684892  | 2.3 %            | 0.0687    |
+| ERC20VotesCheckerFactory                 | -       | -       | 464123  | 1.5 %            | 0.0466    |
+| ERC20VotesInitialVoiceCreditProxyFactory | -       | -       | 482127  | 1.6 %            | 0.0484    |
+| ERC20VotesPolicyFactory                  | -       | -       | 685960  | 2.3 %            | 0.0688    |
+| FreeForAllCheckerFactory                 | -       | -       | 329927  | 1.1 %            | 0.0331    |
+| FreeForAllPolicyFactory                  | -       | -       | 653501  | 2.2 %            | 0.0656    |
+| GitcoinPassportCheckerFactory            | -       | -       | 463912  | 1.5 %            | 0.0466    |
+| GitcoinPassportPolicyFactory             | -       | -       | 687052  | 2.3 %            | 0.0689    |
+| Hasher                                   | -       | -       | 426472  | 1.4 %            | 0.0428    |
+| HatsCheckerFactory                       | -       | -       | 572538  | 1.9 %            | 0.0575    |
+| HatsPolicyFactory                        | -       | -       | 684664  | 2.3 %            | 0.0687    |
+| MACI                                     | 1803354 | 1803438 | 1803408 | 6.0 %            | 0.181     |
+| MerkleProofCheckerFactory                | -       | -       | 491985  | 1.6 %            | 0.0494    |
+| MerkleProofPolicyFactory                 | -       | -       | 686188  | 2.3 %            | 0.0689    |
+| MessageProcessorFactory                  | 1818377 | 1818401 | 1818398 | 6.1 %            | 0.1825    |
+| PollFactory                              | 3720635 | 3720683 | 3720678 | 2.4 %            | 0.3734    |
+| PoseidonT3                               | -       | -       | 2157126 | 7.2 %            | 0.2165    |
+| PoseidonT4                               | -       | -       | 2745274 | 9.2 %            | 0.2755    |
+| PoseidonT5                               | -       | -       | 3569887 | 1.9 %            | 0.3582    |
```

Comment:
Should `PoseidonT5` be `11.9`?
Should `PoseidonT6` be `14.2`?

Same for Linear